### ISSUE TITLE
Two small issues in tutorial

### ISSUE THREAD

**Issue by ynux** [muffin87/helm-tutorial#2]
Thanks for the tutorial.
I ran into these issues:
1 - too low Kubernetes / minikube version for rbac.
Didn't work:

    minikube version: v0.22.1
    Kubernetes v1.7.5 cluster...

    $ kubectl create -f part-02/templates/tiller-rbac.yml
    Error from server (BadRequest): error when creating "part-02/templates/tiller-rbac.yml":       ClusterRoleBinding in version "v1" cannot be handled as a ClusterRoleBinding: no kind "ClusterRoleBinding" is registered for version "rbac.authorization.k8s.io/v1"

Worked with: minikube version: v0.28.2.  Kubernetes v1.10.0 
Probably needs 1.8

2 - timout 

    helm install --name wiki stable/dokuwiki

even though both helm repo update and helm version worked.

**Comment by muffin87** [comment]
Hi @ynux,

thanks for sharing your questions. Please find my answers below.

1) Yes that is right! I used `apiVersion: rbac.authorization.k8s.io/v1` for the ClusterRoleBindng. `v1` was introduced in Kubernetes version 1.8. I tried to highlight the version requirements in [Part I](https://github.com/muffin87/helm-tutorial/blob/master/part-01/README.md) of the tutorial.

```
Note: Since we only need a working Kubernetes API you could also use any other cluster you like. I only tested this tutorial on K8S version 1.9.X, 1.10.X and 1.11.X.
```

My suggestion is that you downgrade the file if needed to `apiVersion: rbac.authorization.k8s.io/v1beta1` that should be available in Kubernetes version 1.6 und 1.7.

2. That is really strange. Never had a timeout when `helm repo update` worked. Maybe your connection was unstable at the moment you executed the command. Have you tried it again? If that happens over and over you could try to add `--debug` option. Maybe Helm tells us what's the problem here.

**Comment by ynux** [comment]
Thanks for the quick response. About the timeout, I guess I did something horrible to my minikube's network before the workshop. So this isn't really a workshop issue, let's close this.

Some forensic notes in case someone is into this.
I checked the tiller pod's logs:

    [tiller] 2018/07/26 06:49:41 preparing install for wiki
    [storage] 2018/07/26 06:49:41 getting release history for "wiki"
    [storage/driver] 2018/07/26 06:50:11 query: failed to query with labels: Get https://10.96.0.1:443.   /api/v1/namespaces/tools/configmaps?labelSelector=NAME%3Dwiki%2COWNER%3DTILLER: dial tcp 10.96.0.1:443: i/o timeout
[tiller] 2018/07/26 06:50:41 failed install prepare step: Get https://10.96.0.1:443/version: dial tcp     10.96.0.1:443: i/o timeout

There seem to be more people hitting this issue, e.g. https://github.com/helm/helm/issues/3347 and https://github.com/helm/helm/issues/4171

      --service-cluster-ip-range=10.96.0.0/12

    kubectl get pod --all-namespaces 
    NAMESPACE     NAME                                    READY     STATUS             RESTARTS   AGE
    kube-system   etcd-minikube                           1/1       Running            0          1d
    kube-system   kube-addon-manager-minikube             1/1       Running            0          1d
    kube-system   kube-apiserver-minikube                 1/1       Running            0          1d
    kube-system   kube-controller-manager-minikube        1/1       Running            0          1d
    kube-system   kube-scheduler-minikube                 1/1       Running            0          1d
    kube-system   kubernetes-dashboard-5498ccf677-rkqbx   0/1       CrashLoopBackOff   14         1d
    kube-system   storage-provisioner                     0/1       CrashLoopBackOff   14         1d
    tools         tiller-deploy-66998d5d74-qv926          1/1       Running            0          1d
